src/content/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent.mdx

```diff
@@ -752,12 +752,26 @@ The Node.js agent integrates with other features to give you observability acros
       </th>
 
       <th>
-        Integration
+        Feature
       </th>
     </tr>
   </thead>
 
   <tbody>
+        <tr>
```

Comment:
I'm not sure if this gets lost. I was envisioning adding another section above like `Datastores` or `Logging Libraries`